cores/arduino/CDCSerialClass.cpp

```diff
@@ -73,7 +100,9 @@ void CDCSerialClass::init(const uint32_t dwBaudRate, const uint8_t modeReg)
      * Empty the Rx buffer but don't touch Tx buffer: it is drained by the
      * LMT one way or another */
     _rx_buffer->tail = _rx_buffer->head;
-
+    
+    mailbox_register(6, cdc_mbox_isr);
```

Comment:
Channel 6 is exposed by the CurieMailbox library. If you use this channel, the Mailbox library and Serial will not work simultaneously.

Can we use channel 7 for both RX and TX?